packages/webdriverio/src/scripts/isElementDisplayed.ts

```diff
@@ -114,25 +114,38 @@ export default function isElementDisplayed (element: Element): boolean {
         return cascadedStylePropertyForElement(parentElement, property)
     }
 
-    function elementHasBoundingBox(element: Element): boolean {
+    function elementSubtreeHasNonZeroDimension({
```

Comment:
It's better to check for Zero dimensions than for non Zero dimensions for the function. Names like these are confusing especially when executed like so !nonZero

Reply:
The diff makes it seem that this is new code, while it isn't: this is an existing function that I modified to check dimensions one by one instead. 
The function is used recursively. I agree that at the top level it would be better to write `elementSubtreeHasZeroDimension()" instead of "!elementSubtreeHasNonZeroDimension()`. But in the recursive algorithm, the original name makes more sense. We could write a new `const elementSubtreeHasZeroDimension = (...args) => !elemelementSubtreeHasNonZeroDimension(...args)` helper, but I feel that would be a bit overkill